src/Codeception/Util/ActionSequence.php

```diff
@@ -54,7 +56,9 @@ public static function build()
 
     public function __call($action, $arguments)
     {
-        $this->actions[$action] = $arguments;
+        $key = "{$this->counter}.$action";
```

Comment:
yes, I agree not a perfect solution. Ok, I think I can rework it.

For output this action sequence should be transformed to a list of actions. So I was thinking to make it as visible array. Right now I think the `ActionSequence` can provide `__toString` and won't be translated to array.